usaspending_api/database_scripts/matview_generator/matview_sql_generator.py

```diff
@@ -371,6 +372,18 @@ def create_monolith_file(sql_json):
     write_sql_file(sql_strings, os.path.join(DEST_FOLDER, sql_json["final_name"]))
 
 
+def add_chunk_strings(sql_json, chunk):
+    chunked_sql_json = copy.deepcopy(sql_json)
+
+    chunk_count = GLOBAL_ARGS.chunks
+
+    if chunk_count > 1:
+        chunked_sql_json["final_name"] += "_{}".format(chunk)
```

Comment:
If you are still working this would be nice to see f-strings here (e.g. `f"_{chunk}"`) and below.